src/addReviewServer.js

```diff
@@ -0,0 +1,17 @@
+import fetch from "node-fetch";
```

Comment:
What version of node do you have on your computer ? Don't think you have to import fetch in version 18 at least.

Reply:
v16.17.0 ... Wops.. Will fix it for the next project. Thank you! :)